sljit_src/sljitLir.c

```diff
@@ -655,7 +657,7 @@ static SLJIT_INLINE sljit_uw compute_next_addr(struct sljit_label *label, struct
 	return result;
 }
 
-#endif /* !SLJIT_CONFIG_X86 */
+#endif /* !CISC */
```

Comment:
would you prefer !RISC or CISC?; apologizes for getting it backwards, I was hoping it will be clearer then to whoever find this function later than it only make sense with RISC since CISC will also need the op to calculate the next address.